open-metadata-implementation/view-services/rex-view/rex-view-api/src/main/java/org/odpi/openmetadata/viewservices/rex/api/rest/RexRelationshipRequestBody.java

```diff
@@ -68,7 +69,6 @@ public String toString()
                 ", platformName=" + platformName +
                 ", relationshipGUID=" + relationshipGUID +
                 ", enterpriseOption=" + enterpriseOption +
-                ", gen=" + gen +
```

Comment:
I assume we use toString for debugging , in that case wouldn't we want all the content like gen and depth etc. As an observation, normally there would be an equals and hashCode associated with a java bean.

Reply:
I have removed gen (completely).